kong/db/schema/metaschema.lua

```diff
@@ -206,6 +206,15 @@ local field_schema = {
       { message = { type = "string", required = true } },
       { removal_in_version = { type = "string", required = true } },
       { old_default = { type = "any", required = false } },
+      { replaced_with = { type = "array", required = false,
+          elements = { type = "record",
+            required = false,
+            fields = {
+              { path = { type = "array", len_min = 1, required = true, elements = { type = "string"}} },
+              { translation = { type = "function", required = false }}
```

Comment:
Here are a couple of naming alternatives:

- transform_function
- mapping_function
- conversion_function
- transformer
- data_mapper
- converter
- processor_function
- value_transform
- format_function

Reply:
or just func? like in shorthnds?

Reply:
I changed it to `reverse_mapping_function`. WDYT? 🤔

Reply:
ok, seems like we want to everytime different name for custom `func`. I checked and there are `func`, `fn`, `on_write`, `on_read`, `check` and `custom_validator`. And now we are introducing `reverse_mapping_function`.